rgw/v2/tests/s3_swift/reusables/bucket_notification.py

```diff
@@ -369,9 +372,15 @@ def verify_event_record(event_type, bucket, event_record_path, ceph_version):
                 raise EventRecordDataError("awsRegion not in event record")
 
     if notifications_received is False:
-        raise EventRecordDataError(
-            f"Notifications not received for event type {event_type}"
-        )
+        if config.test_ops.get("test_delete_object_sync_archive", False):
```

Comment:
@viduship as per the above change in method arguments, please modify this as below
if config and config.test_ops.get("test_delete_object_sync_archive", False):